content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -314,6 +314,7 @@ kubectl exec my-pod -- ls /                         # Run command in existing po
 kubectl exec --stdin --tty my-pod -- /bin/sh        # Interactive shell access to a running pod (1 container case) 
 kubectl exec my-pod -c my-container -- ls /         # Run command in existing pod (multi-container case)
 kubectl top pod POD_NAME --containers               # Show metrics for a given pod and its containers
+kubectl top pod POD_NAME --sort-type=cpu            # Show metrics for a given pod and sort it by 'cpu' or 'memory'
```

Comment:
Should this be --sort-by instead of --sort-type?  
See https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#top

Reply:
Yes, that's true. My mistake. I will fix it.